src/components/Search/SearchBar/SearchPopover.tsx

```diff
@@ -0,0 +1,119 @@
+import { useRef, useState } from "react";
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import IconButton from "@leafygreen-ui/icon-button";
+import { palette } from "@leafygreen-ui/palette";
+import Popover from "@leafygreen-ui/popover";
+import { Body, BodyProps } from "@leafygreen-ui/typography";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+import { useOnClickOutside } from "hooks";
+
+const { white, blue, gray } = palette;
+
+interface SearchPopoverProps {
+  disabled: boolean;
+  suggestions: string[];
+  onClick?: (suggestion: string) => void;
+}
+
+const SearchPopover: React.FC<SearchPopoverProps> = ({
+  disabled,
+  suggestions,
+  onClick,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const popoverRef = useRef<HTMLDivElement>(null);
+
+  useOnClickOutside([buttonRef, popoverRef], () => {
+    setIsOpen(false);
+  });
+
+  const handleClick = (suggestion: string) => {
+    setIsOpen(!isOpen);
+    onClick?.(suggestion);
+  };
+
+  return (
+    <SearchPopoverBox disabled={disabled}>
+      <IconButton
+        ref={buttonRef}
+        aria-labelledby="View search suggestions"
+        data-cy="search-suggestion-button"
+        disabled={disabled}
+        onClick={() => setIsOpen(!isOpen)}
+      >
+        <>
+          <Icon fill={gray.base} glyph="Bulb" />
+          <Icon fill={gray.base} glyph="CaretDown" />
+        </>
+        <Popover active={isOpen} data-cy="search-suggestion-popover">
+          <div ref={popoverRef}>
+            <StyledCard>
+              {suggestions.length > 0 ? (
+                suggestions.map((s) => (
+                  <SearchSuggestion
+                    key={s}
+                    onClick={() => handleClick(s)}
+                    role="button"
+                  >
+                    {s}
+                  </SearchSuggestion>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

There are also a few missing properties that are required when making a div a button-like element.

Reply:
Here's an example of how it would work when tabindex and accessibility attributes are applied properly. (Although I couldn't figure out how to get it to autofocus on the first button immediately after opening the list but I didn't try too hard)
<img width="502" alt="image" src="https://github.com/evergreen-ci/parsley/assets/4605522/d4805437-5467-4388-8730-99745a4646f8">

Reply:
I just changed it into a real button since that seemed to make more sense! I'll make a ticket for the up/down arrow functionality. At that time it may also make sense to move to LeafyGreen's `InputOption` component instead 🫨